### PR TITLE
Test againt Ruby 2.2 on Travis CI

### PR DESCRIPTION
**Pull request by tricknotes**

### CODE REVIEW

**Comment by mkdynamic** [comment]
Thanks!

File: README.md [hunk 1]
```diff
@@ -158,6 +158,7 @@ If you use the server-side flow, Facebook will give you back a longer lived acce
 
 Actively tested with the following Ruby versions:
 
+- MRI 2.2.0
 - MRI 2.1.0
```
Comment (simi) [inline README.md]: Aren't we testing only the latest minor version? For example for 2.1 I can see there's 2.1.5 tested on CI.

Comment (tricknotes) [inline README.md]: Seems right.
Currently this version is incorrect.

But patch version in README is hardly to maintain.
Might it be `2.1` or `2.1.x` ?

Comment (simi) [inline README.md]: I think 2.1 an 2.2 is enough.

Comment (tricknotes) [inline README.md]: Agreed.
I send a new PR: https://github.com/mkdynamic/omniauth-facebook/pull/226

**Pull request merged**